src/Extension/CoreExtension.php

```diff
@@ -553,7 +553,7 @@ public function convertDate($date = null, $timezone = null)
     public static function replace($str, $from): string
     {
         if (!is_iterable($from)) {
-            throw new RuntimeError(\sprintf('The "replace" filter expects a sequence/mapping or "Traversable" as replace values, got "%s".', \is_object($from) ? \get_class($from) : \gettype($from)));
+            throw new RuntimeError(\sprintf('The "replace" filter expects a sequence or a mapping, got "%s".', \is_object($from) ? \get_class($from) : \gettype($from)));
```

Comment:
While we're at it, shall we also pick a consistent way of outputting the type of the invalid parameter?

```suggestion
            throw new RuntimeError(\sprintf('The "replace" filter expects a sequence or a mapping, got "%s".', get_debug_type($from)));
```